### PR TITLE
feat: added Metis logo

### PR DESCRIPTION
**Pull request by america-nishi**
Added metis logo with transparent background.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/src/favicon.svg" />
+    <link rel="icon" type="image/png" href="/public/favicon.png" />
```
Comment (AlefrankM) [inline index.html:5]: It's looks better like
```suggestion
    <link rel="icon" type="image/png" href="/favicon.png" />
```

File: src/pages/login-page/LoginPage.tsx [hunk 2]
```diff
@@ -86,7 +86,7 @@ const LoginPage = () => {
       <Box height="100vh" className={classes.wrapper}>
         <Container maxWidth="xl" component="main" className={classes.container}>
           <Box component="form" noValidate maxWidth="md">
-            <Box component="img" src={SquareGroup} alt="login" className={classes.image} />
+            <Box component="img" src={MetisLogo} alt="login" className={classes.image} />
```
Comment (AlefrankM) [inline src/pages/login-page/LoginPage.tsx:89]: I'm watching the result and think so we will change to svg format or put a image with major quality. The image looks up pixeled

**Review by AlefrankM — APPROVED** [review]
The result could be better

File: index.html (hunk 1 above)
Comment (josetheis-stackit) [reply index.html:5]: Can you do this? Is there any extra config we need to do so this can actually work?

Comment (AlefrankM) [reply index.html:5]: Yes, it's default behavior. Equally, I tested and works!

**Review by AlefrankM — APPROVED** [review]
LGTM

**Review by mmiranda-stackit — COMMENTED** [review]
Just being a little curious, do we need all those 4 images in our code? Or just two of them?

**Pull request merged**